packages/api/cms-api/src/dam/images/images.controller.ts

```diff
@@ -41,6 +42,7 @@ const smartImageUrl = `:fileId/crop::focalPoint([A-Z]{5,9})/resize::resizeWidth:
 const focusImageUrl = `:fileId/crop::cropWidth::cropHeight::focalPoint::cropX::cropY/resize::resizeWidth::resizeHeight/:filename`;
 
 @Controller("dam/images")
+@RequiredPermission(["dam"], { skipScopeCheck: true })
```

Comment:
Are we okay with allowing accessing the preview URLs for all scopes? Could this be a security issue?

Reply:
The scope in this Controller are checked manually in the relevant actions. I added a comment: 6bb0de8e1f45278a43a2dda119e25bcf73ca6940

However, when I find time I want to check if we can use the provided system instead the manual check.